tabbycat/motions/templates/show.html

```diff
@@ -75,7 +75,13 @@
         {% if infos|length == 1 %}
 
           <div class="modal-body d-flex align-items-center justify-content-center text-center">
-            <small>{{ infos.0.motion.info_slide|linebreaks }}</small>
+            <small>
+              {% if infos.0.motion.info_slide|slice:":3" == "<p>" %}
```

Comment:
Is this for backwards compatibility? Or, could we assume all motions going forward using rich text and can be output without using `|linebreaks`? (I may be totally forgetting how this part of django formatting works).

Reply:
Yes, but there wouldn't be a way to determine if the motion was created when rich text was added or not, so we can test the text itself for the tag. Also, the presets don't have rich text motions.